api/src/main/java/keywhiz/api/model/Group.java

```diff
@@ -71,6 +79,27 @@ public Group(@JsonProperty("id") long id,
     this.updatedAt = updatedAt;
     this.updatedBy = nullToEmpty(updatedBy);
     this.metadata = ImmutableMap.copyOf(metadata == null ? ImmutableMap.of() : metadata);
+    this.owner = owner;
+  }
+
+  public Group(long id,
```

Comment:
does this need Json annotations too?

Reply:
Interestingly, it had them originally, which caused Jackson to fail.

Jackson expects there to be a single constructor or factory method annotated with @JsonCreator, but if you leave that annotation off (which we were doing previously) then it scans the class for a constructor whose arguments have JSON-ish annotations. If you have multiple such constructors, it complains that it doesn't know which one to use and fails.

So, since this constructor is only intended for backwards compatibility, it doesn't have any JSON annotations because Jackson will never invoke it. I've also explicitly marked the other constructor with @JsonCreator so that we're a bit more well behaved.